### DISCUSSION TITLE
All default colour hex codes and ColorizationBalance values for Windows Vista beta 2 5370 up to Windows 8.1

### DISCUSSION THREAD

**Discussion by TheInsane101** [Maplespe/DWMBlurGlass#309] [Show and tell]
Collected all the values from regedit from Windows 7 Milestone 3 6801, 7 SP1 (vruh did 7 though, not me), 8 (w/ glass8 installed) and 8.1 (w/ glass8 installed, so it's pretty much the same as the Release Preview version), including all regional Windows 7 themes. Pack includes text files listing all the colour values together with the config file equivalents for DWMBlurGlass 2.2. 

Theme files, colour registry keys and themecpl files not included. Don't have time to add them now; _might_ add them in a later update.

Windows Vista's colours were collected by someone else in the "7 Aero research" thread and I ported them over too. However, you need to use Vista's configs with Vruh's fork of 2.2 in order to get Vista's Aero, as 2.1 replaced the "Blur" shader with 7's shader but without the Balance effects. Vruh's fork is in his own comment down below this post.

BONUS: You will find folders with "Vista to 7" etc in the title, which are collections of Vista's configs converted into as if they had existed in Windows 7 itself!

Hope you enjoy!

[Windows Vista beta 2 - 8.1 Aero configs for 2.2.zip](https://github.com/user-attachments/files/15872758/Windows.Vista.beta.2.-.8.1.Aero.configs.for.2.2.zip)

**Comment by funn-hue** [comment]
This is gonna be extremely useful. Thanks

**Reply by TheInsane101** [reply]
Glad I could help! You'll need to make the matching .theme files yourself though, for now.

**Comment by Randomuser8219** [comment]
You can get the Vista style Aero effect by using DWMBlurGlass 2.0.1's Aero shader

**Comment by kfh83** [comment]
> Collected all the values from regedit from Windows 7 PDC 2008, 7 SP1 (vruh did 7 though, not me), 8 (w/ glass8 installed) and 8.1 (w/ glass8 installed, so it's pretty much the same as the Release Preview version), including all regional Windows 7 themes. Pack includes text files listing all the colour values together with the config file equivalents for DWMBlurGlass 2.2.
> 
> Theme files, colour registry keys and themecpl files not included. Don't have time to add them now; _might_ add them in a later update.
> 
> Windows Vista not included because DWMBG has not been prioritised to mimic Vista's Aero, despite it being simpler unfortunately.
> 
> Hope you enjoy!
> 
> [Ultimate Default Colour collection for Windows 7 PDC 2008 - Windows 8.1.zip](https://github.com/Maplespe/DWMBlurGlass/files/15209451/Ultimate.Default.Colour.collection.for.Windows.7.PDC.2008.-.Windows.8.1.zip)

I can probably PR the blur shader back to it's 2.0.1 specs and then you'll have vista colorization again. Matter of fact, this is what i did on my fork, but i also made it way more hardcoded (inactive color is fixed to be the same as the active, and always 50% opacity of active). I'm not sure what happened in 2.1.0 but the Blur shader got updated to use multiply blending, so it looks a lot more like the aero shader (... without any of the fancy blending that it now has, or any of the inactive hacks it had in 2.1.0/2.1.1)

[427.zip](https://github.com/Maplespe/DWMBlurGlass/files/15210341/427.zip)

Another side note, this fork has the 7 captions button option gimped to only render button glow and do nothing else, so you'll have to use the windhawk captions mod to get proper button sizes and text positioning. Furthermore, it's older than the new occlusion stuff, but i think performance should be fine regardless on the blur shader.

**Reply by TheInsane101** [reply]
Your fork works amazingly well! You also fixed the caption button misplacement bug! Thank you!

**Reply by Skyyblaze** [reply]
Sadly with this fork the restoring Windows 7 caption height shrinking doesn't seem to work.

**Reply by ImSwordQueen** [reply]
> Another side note, this fork has the 7 captions button option gimped to only render button glow and do nothing else

🤦

**Reply by Skyyblaze** [reply]
Sorry my bad, my eyes skipped over this!

**Reply by TheInsane101** [reply]
You need to use the Restore 7 Sizes mod for Windhawk together with the fork.

**Reply by InsaneGal** [reply]
> You need to use the Restore 7 Sizes mod for Windhawk together with the fork.

So this replicates the Vista aero effect? Nice, I tried looking for that mod in Windhawk though and can't find it. Is that the specific name of it?

**Reply by TheInsane101** [reply]
[Here's where the buttons are](https://winclassic.net/thread/2186/dwm-windows-frame-hooking-mods)

**Reply by paperboi31** [reply]
for some reason that mod didnt work? still just the normal windows 10 sized buttons with the vista theme

**Comment by TheInsane101** [comment]
> Sadly with this fork the restoring Windows 7 caption height shrinking doesn't seem to work.

Odd, it works for me

![Screenshot (1710)](https://github.com/Maplespe/DWMBlurGlass/assets/104862146/3cd77013-4de6-448a-b85f-d58a7820e352)

**Reply by kfh83** [reply]
They meant that the Restore Win7 style title bar button size option doesn't resize the buttons (they read over my comment where i mentioned it being strictly glow only)
